templates/highlighter.css

```diff
@@ -0,0 +1,12 @@
+.__popcode-highlighter {
+  position:absolute;
+  z-index: 10000000000;
+  border:1px solid rgba(238, 238, 48, 0.2);
```

Comment:
Is this accessible?

Reply:
@kaylee42 I think we should talk about this one too. 

Its currently based on the cloud9 color scheme but thimble might better jumping off point https://thimble.mozilla.org